### DISCUSSION TITLE
Option to set state-specific border-radius inside nested Accordion

### DISCUSSION THREAD

**Discussion by AF1QUE** [elementor/elementor#24990] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Problem: I cannot set the different border-radius for each state in the accordion widget; for instance, changing the border-radius value in the active state also sets the same value for other states like Normal/Active and vice versa.

Check the screenshot for better further explanation

<img width="1440" alt="Screenshot 2024-01-31 at 7 13 46 AM" src="https://github.com/elementor/elementor/assets/46385375/60b19f1d-ebbb-4d5a-8b82-129c818b5df2">

### Describe the solution you'd like.

Solution: Give users the ability to set different border-radius values for each state individually.

### Additional context

The feature should work for any widgets, not only for Accordion; we would very much like to have a customization option over the border-radius for any widget Elementor offers. 

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by q1LLa** [comment]
I support you, I have the same problem. Have you found a solution?
